fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/systems.cfg

```
@@ -777,3 +797,5 @@ LocalVar.74 = A380X_RMP_3_NAV_SEL
 LocalVarDefault.74 = 0
 LocalVar.75 = A380X_RMP_3_INT_RAD
 LocalVarDefault.75 = 1
+LocalVar.76 = A380X_CABIN_HIDDEN
+LocalVarDefault.76 = 0
```

```suggestion
LocalVar.76 = A380X_CABIN_HIDDEN
LocalVarDefault.76 = 0
```

Don't think this belongs here

---

Was this part of another PR? It's gone now, after merging with master